src/wasm-config.h.in

```diff
@@ -102,6 +110,12 @@
 #define WASM_LIKELY(x) (x)
 #define WASM_PRINTF_FORMAT(format_arg, first_arg)
 
+/* WABT_BUILTIN_UNREACHABLE - Expands to an expression which states
+ * that it is undefined behavior for the compiler to reach this
+ * point.  Otherwise is not defined.
+ */
```

Comment:
Cleaned this up and now using WABT_UNREACHABLE. Also added else case to generate an error.